packages/chart/src/components/Legend/Legend.Component.tsx

```diff
@@ -22,29 +24,30 @@ export interface LegendProps {
   ref: React.Ref<() => void>
   seriesHighlight: string[]
   skipId: string
+  dimensions: [string, string]
```

Comment:
Add comment above describing what dimensions are for.

Reply:
Fixed